src/sentry/services/hybrid_cloud/organization_actions/impl.py

```diff
@@ -0,0 +1,44 @@
+from typing import TypedDict
+
+from django.db import transaction
+from django.db.models.expressions import CombinedExpression
+
+from sentry.models import Organization, OrganizationStatus
+
+
+class OrganizationCreateAndUpdateOptions(TypedDict, total=False):
+    name: str
+    slug: str
+    status: OrganizationStatus
+    flags: CombinedExpression
```

Comment:
Hmm, will this type be serializable in the end?

Reply:
Does it have to be? We're really only using `name` and `slug` for organization creation as far as I know. Are there any creation flows more complex than this that create an org with non-default flags?

Reply:
Ah I see you are correct.

Reply:
Fwiw, the RPC side will have a trimmed down version of this for org creation, with the addition of any user data we need to sync over